spec/feature/posts_index_spec.rb

```diff
@@ -1,36 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'User post index page', type: :feature do
-  let!(:user) { User.create(name: 'Sarkodie', photo: '/path/to/photo1.jpg', post_counter: 10) }
+  let!(:user) { User.create(name: 'Sarkodie', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', posts_counter: 10) }
 
   before do
     5.times do
       Post.create(
         title: 'Post title',
         text: 'Post text',
         author_id: user.id,
-        comments_counter: 3,
-        likes_counter: 7
+        comments_counter: 2,
+        likes_counter: 1
       )
     end
     visit user_posts_path(user)
   end
 
-  it 'displays user profile information' do
-    expect(page).to have_css("img[src='#{user.photo}'][alt='user photo']")
-    expect(page).to have_content(user.name)
-    expect(page).to have_content("Number of posts: #{user.post_counter}")
-  end
-
   it 'displays each post with relevant information' do
     @posts = user.posts
 
     @posts.each do |post|
-      expect(page).to have_content(post.title)
```

Comment:
- [ ] You are currently missing:
- Test that asserts you can see the number of posts the user has written.
- Test that asserts you can see a post's title.
- Test that asserts you can see some of the post's body.
Kindly add the missing tests